.github/workflows/assets.yml

```diff
@@ -31,12 +31,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          RAW_TAG="${{ inputs.tag_ref }}"
+          TAG="${RAW_TAG#refs/tags/}"
```

Comment:
We don't do like this on any other workflow, why not use like we have on master: https://github.com/lf-edge/eve/blob/master/.github/workflows/assets.yml#L75

`echo "TAG=$(git describe --always --tags | grep -E '[0-9]+\.[0-9]+\.[0-9]' || echo snapshot)" >> "$GITHUB_ENV"`

Reply:
But that moment we don't have the tag checkout out yet...